sdk/PrebidMobile/PBAdUnit.h

```diff
@@ -43,6 +43,8 @@
  */
 @property (nonatomic, readonly) NSArray<CGSize> *__nullable adSizes;
 
+@property (nonatomic, readwrite) BOOL isSecure;
+
```

Comment:
The feature here is by default use secure connection for all requests, allow MoPub ad server clients to override to load non-secure content. Adding `secure:1` is just a open rtb implementation, should not be adding this setting per ad unit.

Reply:
removed this and added to the PreBidServerAdapter parameters

Reply:
isSecure should be removed.